### ISSUE TITLE
High cpu usage after enabling x-ray tracing

### ISSUE THREAD

**Issue by ricardorosg** [krakendio/krakend-ce#380]
Hello, after enabling x-ray tracing the cpu of the container were i'm running krakend increases to 100% inmediatlly.
I'm testing it in a development environtment with a very low traffic.
I think I dont have nothing strange in my config (mainly copied from the docs).

```
"extra_config": {
    "github_com/devopsfaith/krakend-opencensus": {
      "sample_rate": 10,
      "reporting_period": 30,
      "enabled_layers": {
        "backend": true,
        "router": true,
        "pipe": false
      },
      "exporters": {
        "xray": {
          "version": "latest",
          "region": "eu-west-1",
          "use_env": true
        },
        "influxdb": {
          "address": "influx",
          "db": "krakend",
          "timeout": "2s",
          "username": "admin",
          "password": "admin"
        }
      }
    },
    "github_com/devopsfaith/krakend-metrics": {
      "collection_time": "30s",
      "listen_address": "127.0.0.1:8090"
    },
    "github_com/letgoapp/krakend-influx": {
      "address": "influx",
      "ttl": "30s",
      "buffer_size": "{{ .service.buffer_size }}",
      "db": "krakend"
    }
  }
```
![Captura de pantalla_2021-11-04_18-05-25](https://user-images.githubusercontent.com/18736596/140385899-b59c221a-e7c6-4bfa-8491-f4a7f08970e6.png)

**Comment by alombarte** [comment]
Hi, by seeing the configuration only it's complicated to do a full diagnosis, but some thoughts below...

My first question would be if you are running KrakenD in a micro/small instance. Please share the specs behind this installation. In addition to that:
- If your machine has a single CPU, it will compete harder against the processing of the traffic.
- Telemetry isn't CPU free. It has to gather, process and push a LOT of data.
- Plus, you have not one but THREE telemetry exporters enabled simultaneously: X-ray, Opencensus Influx, Native Influx
- The reporting period of 30 seconds might be too high and inadequate. 
- The timeout of 30s for the native InfluxDB is crazy high
- You don't show the buffer_size

TL;DR: Disable the two InfluxDB reporters and see the CPU behavior alone, and let's see from here. Share also the logs.
